services/source-debugger/eventUploader.go

```diff
@@ -22,25 +22,11 @@ type GatewayEventBatchT struct {
 	eventBatch string
 }
 
-//EventUploadT is a structure to hold actual event data
-type EventUploadT struct {
-	Event             string      `json:"event,omitempty"`
-	Integrations      interface{} `json:"integrations,omitempty"`
-	Properties        interface{} `json:"properties,omitempty"`
-	OriginalTimestamp string      `json:"originalTimestamp,omitempty"`
-	AnonymousID       string      `json:"anonymousId,omitempty"`
-	UserID            string      `json:"userId,omitempty"`
-	MessageID         string      `json:"messageId,omitempty"`
-	Name              string      `json:"name,omitempty"`
-	SentAt            string      `json:"sentAt,omitempty"`
-	Type              string      `json:"type,omitempty"`
-}
-
 //EventUploadBatchT is a structure to hold batch of events
 type EventUploadBatchT struct {
 	WriteKey   string
 	ReceivedAt string
-	Batch      []EventUploadT
+	Batch      []map[string]interface{}
```

Comment:
Define a type for EventUpload.
`type EventUploadT map[string]interface{}`

Reply:
Makes all further references more readable.

Reply:
done.